### DISCUSSION TITLE
Redirect "edit" links after default branch is renamed

### DISCUSSION THREAD

**Discussion by targos** [community/community#28849] [Other Feature Feedback, Questions, & Ideas]
Hello, in projects like Node.js, where a static documentation website is build and published for each released version, we have an issue with "Edit on GitHub" links (available in the [Options dropdown](https://nodejs.org/dist/v18.5.0/docs/api/async_hooks.html)), that stopped working after we renamed the `master` branch to `main`.

Would it be possible to automatically redirect users from links like https://github.com/nodejs/node/edit/master/doc/api/async_hooks.md to https://github.com/nodejs/node/edit/main/doc/api/async_hooks.md ?

**Comment by rpriyanshu9** [comment]
I guess this issue might have been fixed in the later versions. In the latest LTS and normal versions this option works fine. You can verify here https://nodejs.org/en/docs/
